docs/review_process.md

```diff
@@ -0,0 +1,136 @@
+Right now, neither GitHub itself nor conan-center-bot notify about merge conflict, so it's contributor responsibility to periodically check for the conflicts.
+Obviously, PRs that have merge conflicts are never merged, and all the conflicts has to be resolved first.
+
+## draft
+
+Draft pull requests are also never merged. The same applies for `WIP` keyword (stands for `Work in Progress`) in the title - GitHub considers them drafts as well.
+Once you're done with your changes, remember to change from draft and remove WIP keyword from the title.
+
+## getting PR reviewed
+
+Each PR should be reviewed by several reviewers before it could be merged. It cannot be just reviews from random people, we have two categories of the reviewers:
+
+### official reviewers
+
+The list includes only official conan developers (JFrog employees):
+
+- [@memsharded](https://github.com/memsharded)
+- [@lasote](https://github.com/lasote)
+- [@danimtb](https://github.com/danimtb)
+- [@jgsogo](https://github.com/jgsogo)
+- [@czoido](https://github.com/czoido)
+- [@solvingj](https://github.com/solvingj)
+- [@sse4](https://github.com/sse4)
+- [@uilianries](https://github.com/uilianries)
+
+### community reviewers
+
+The list includes conan center index contributors who are very active and proven to be trusted - they frequently submitted pull requests and provide their own useful reviews:
+
+- [@madebr](https://github.com/madebr)
+- [@SpaceIm](https://github.com/SpaceIm)
+- [@ericLemanissier](https://github.com/ericLemanissier)
+- [@prince-chrismc](https://github.com/prince-chrismc)
+- [@Croydon](https://github.com/Croydon)
+- [@intelligide](https://github.com/intelligide)
+- [@theirix](https://github.com/theirix)
+- [@gocarlos](https://github.com/gocarlos)
+
+The list is not constant and rolled periodically, as people's involvement change.
+That also means you can be included to that list as well - submit PRs and provide reviews in order to be included to the list above.
+
+### Rule of 3 reviews
+
+At least 3 reviews are required for an approval, and at least one of them has to be from the official reviewers.
+So, it might be 1 official + 2 community, or 3 official, but it couldn't be just 3 community reviews.
+
+### Reviews from others
+
+While reviews from contributors who don't belong neither to official reviewers nor to community reviewers are not counted by the bot, they are still valuable and very helpful, so do not hesistate to provide your reviews.
+
+### Addressing review comments
+
+Please ensure to address the review comments and respond to them in order to get your PR approved and finally merged.
+It doesn't always mean accepting all the suggestions, but at least providing a response, so people can understand your position.
+
+## Automatic Merges
+
+The bot runs Automatic Merges every 30 minutes. By now, it may only merge a single PR at one shot, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
```

Comment:
```suggestion
The bot runs Automatic Merges every 30 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
```